libcnb-data/src/buildpack.rs

```diff
@@ -226,25 +234,28 @@ libcnb_newtype!(
     /// let stack_id: StackId = stack_id!("heroku-20");
     /// ```
     stack_id,
-    /// buildpack.toml Stack Id. This is a newtype wrapper around a String.
-    /// It MUST only contain numbers, letters, and the characters ., /, and -.
-    /// or be `*`.
+    /// The ID of a stack.
+    ///
+    /// It MUST only contain numbers, letters, and the characters `.`, `/`, and `-`.
```

Comment:
This should technically have `*`, but might as well omit for now given it will be true shortly :-)